frontend.py

```diff
@@ -0,0 +1,16 @@
+from converter import CsvToXmlConverter
+import PySimpleGUIQt as sg
+
+
+class GuiFrontend:
+    def __init__(self):
+        self.layout = [[sg.Text('Укажите путь до файла'), sg.InputText()],
```

Comment:
I'd format it for more readability.

```suggestion
        self.layout = [
            [
                sg.Text('Укажите путь до файла'),
                sg.InputText(),
            ],
            [
                sg.Button('ОК'),
            ],
        ]
```